ros_bt_py/ros_bt_py/tree_manager.py

```diff
@@ -964,11 +969,7 @@ def control_execution(  # noqa: C901 # TODO: Remove this and simplfy the method.
             try:
                 root = self.find_root()
                 if root:
-                    with self._state_lock:
-                        self._setting_up = True
                     root.setup()
-                    with self._state_lock:
-                        self._setting_up = False
```

Comment:
Are we just not locking on setup anymore?

Reply:
As there are no read accesses to this variable it did not seemed to be used before.